packages/cli/src/commands/dbCommands/__tests__/dbCommands.test.js

```diff
@@ -37,27 +37,27 @@ describe('db commands', () => {
   it('runs the command as expected', async () => {
     await up.handler({ dbClient: true, schema })
     expect(runCommandTask.mock.results[0].value).toEqual([
-      `yarn prisma migrate up --experimental --create-db --schema=${schema}`,
+      `yarn prisma migrate up --experimental --create-db --schema="${schema}"`,
```

Comment:
Hmmm... shouldn't this actually be `--schema="./api/db/schema.prisma"`

Reply:
Ok, I will revisit.

Reply:
Oh, I'm dumb.